internal/model/db.go

```diff
@@ -0,0 +1,12 @@
+package model
```

Comment:
Database and models are linked together, but they are not the same

Im not sure, that db should stay in models directory